<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowPlantCare</class>
 <widget class="QMainWindow" name="MainWindowPlantCare">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1219</width>
    <height>878</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
    <bold>true</bold>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>v1.0</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhNone</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolBox" name="toolBox">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>260</y>
      <width>751</width>
      <height>461</height>
     </rect>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="toolBoxPage1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>751</width>
       <height>377</height>
      </rect>
     </property>
     <attribute name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/images/user.png</normaloff>:/images/user.png</iconset>
     </attribute>
     <attribute name="label">
      <string>Login as User</string>
     </attribute>
     <widget class="QFrame" name="loginBox">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>40</y>
        <width>751</width>
        <height>341</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
 
QGroupBox {

background-color: rgb(38, 162, 105);
font: 700 12pt;
color: rgb(255, 255, 255);
border:0px solid green;
border-radius: 100px;
background-image: url(:/images/beauty.png);
 

}

 QLineEdit {
  border:3px solid gray;
  
  
  background: rgb(255, 255, 255);
  font: 700 12pt ;
  color: rgb(0, 0, 0);;
}</string>
      </property>
      <widget class="QPushButton" name="pushButton_login">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>220</y>
         <width>111</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">


 QPushButton {
  border:3px solid solid gray;
  border-radius: 10px;
  padding: 0 8px;
  background: green;
 font: 700 12pt ;
 color: rgb(255, 255, 255);
   
}</string>
       </property>
       <property name="text">
        <string>Submit</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_login_3">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>220</y>
         <width>111</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"> 


 QPushButton {
  border:3px solid solid gray;
  border-radius: 10px;
  padding: 0 8px;
  background: green;
 font: 700 12pt ;
 color: rgb(255, 255, 255);
   
}</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_password">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>140</y>
         <width>321</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
 QLineEdit {
  border:3px solid solid gray;
  border-radius: 10px;
  padding: 12px 12px;
  background: rgb(255, 255, 255);
  font: 700 12pt ;
  color: rgb(0, 0, 0);;
}</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>Enter your password</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_username">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>60</y>
         <width>321</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"> QLineEdit {
  border:3px solid solid gray;
  border-radius: 10px;
  padding: 12px 12px;
  background: rgb(255, 255, 255);
  font: 700 12pt ;
  color: rgb(0, 0, 0);;
}</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>Enter your username</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>10</y>
         <width>171</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
QPushButton{
                    color: rgb(255, 255, 255);
                    border:0px solid green;
                    border-radius: 8px;
                    padding: 0 8px;
                    background-color: rgb(0, 90, 0);
                    font: 600 11pt ;

}

 </string>
       </property>
       <property name="text">
        <string> User Login</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="toolBoxPage2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>751</width>
       <height>377</height>
      </rect>
     </property>
     <attribute name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/images/user.png</normaloff>:/images/user.png</iconset>
     </attribute>
     <attribute name="label">
      <string>Login as Administrator</string>
     </attribute>
     <widget class="QFrame" name="loginBox_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>751</width>
        <height>341</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
 

QGroupBox {
 

font: 700 12pt;
color: rgb(255, 255, 255);
border:0px solid green;
border-radius: 100px;
background-image: url(:/images/beauty.png);
 

}

 QLineEdit {
  border:3px solid gray;
  
  
  background: rgb(255, 255, 255);
  font: 700 12pt ;
  color: rgb(0, 0, 0);;
}</string>
      </property>
      <widget class="QPushButton" name="btn_Admin_login">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>220</y>
         <width>111</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"> 


 QPushButton {
  border:3px solid solid gray;
  border-radius: 10px;
  padding: 0 8px;
  background: green;
 font: 700 12pt ;
 color: rgb(255, 255, 255);
   
}</string>
       </property>
       <property name="text">
        <string>Submit</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_Admin_Cancel">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>220</y>
         <width>111</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"> 


 QPushButton {
  border:3px solid solid gray;
  border-radius: 10px;
  padding: 0 8px;
  background: green;
 font: 700 12pt ;
 color: rgb(255, 255, 255);
   
}</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_password_2">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>140</y>
         <width>321</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
 QLineEdit {
  border:3px solid rgb(58, 169, 200);
  border-radius: 10px;
  padding: 12px 12px;
  background: rgb(255, 255, 255);
  font: 700 12pt ;
  color: rgb(0, 0, 0);;
}</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>Enter your password</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_username_2">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>60</y>
         <width>321</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"> QLineEdit {
  border:3px solid  rgb(58, 169, 200);
  border-radius: 10px;
  padding: 12px 12px;
  background: rgb(255, 255, 255);
  font: 700 12pt ;
  color: rgb(0, 0, 0);;
} </string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>Enter your username</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>10</y>
         <width>171</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
QPushButton{
                    color: rgb(255, 255, 255);
                    border:0px solid green;
                    border-radius: 8px;
                    padding: 0 8px;
                    background-color: rgb(0, 90, 0);
                    font: 600 11pt ;

}

 </string>
       </property>
       <property name="text">
        <string> Admin Login</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1219</width>
      <height>101</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>101</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">  QLabel {
  background-color: rgb(0, 85, 0);
  border:3px solid white;
  border-radius: 10px;
  padding: 0 10px;
  
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>731</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton{
                    color: rgb(255, 255, 255);
                    border:3px solid green;
                    border-radius: 8px;
                    padding: 0 8px;
   
                    font: 700 13pt ;

}

 </string>
    </property>
    <property name="text">
     <string>Welcome To The Rose Garden Nursery</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>160</y>
      <width>481</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton{
                    color: rgb(255, 255, 255);
                    border:3px solid green;
                    border-radius: 8px;
                    padding: 0 8px;
                    background-color: rgb(0, 85, 0);
                    font: 600 11pt ;

}

 </string>
    </property>
    <property name="text">
     <string>Please select your account type</string>
    </property>
   </widget>
   <widget class="QLabel" name="CopyRight_line">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>750</y>
      <width>1791</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>41</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
    
	font: 9pt &quot;Segoe UI&quot;;
	background-color: rgb(0, 0, 0);
	
	color: rgb(255, 255, 255);

}</string>
    </property>
    <property name="text">
     <string>      Rose Garden Plant Care System | Copyright © 2024 |Developed by Rabson Innocent Phiri</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1219</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>pushButton</tabstop>
  <tabstop>toolBox</tabstop>
  <tabstop>pushButton_login_3</tabstop>
  <tabstop>lineEdit_password</tabstop>
  <tabstop>lineEdit_username</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>btn_Admin_login</tabstop>
  <tabstop>btn_Admin_Cancel</tabstop>
  <tabstop>lineEdit_password_2</tabstop>
  <tabstop>lineEdit_username_2</tabstop>
  <tabstop>pushButton_4</tabstop>
  <tabstop>pushButton_login</tabstop>
  <tabstop>pushButton_2</tabstop>
 </tabstops>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
